### PR TITLE
Use a shared actions for TF output rendering

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

File: .github/workflows/presubmit-build.yaml [hunk 1]
```diff
@@ -79,20 +79,18 @@ jobs:
           echo ::endgroup::
           terraform apply -auto-approve "-target=module.${{ matrix.imageName }}" -json | tee /tmp/${{ matrix.imageName }}.tf.json | jq -r '.["@message"]'
 
-      - name: Collect diagnostics and upload
+      - name: Collect TF diagnostics and upload
+        if: ${{ always() }}
+        uses: mattmoor/actions-1/terraform-diag@add-terraform-diag
```
Comment (mattmoor) [inline .github/workflows/presubmit-build.yaml:84]: DO NOT SUBMIT: https://github.com/chainguard-dev/actions/pull/341

**Comment by mattmoor** [comment]
Pulumi is failing at head due to Python 3.12, so my plan is to TBR this around that failing leg, but ironically that is useful to help illustrate what this PR does!

**Review by imjasonh — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

**Comment by mattmoor** [comment]
Example of the new output: 
image

**Comment by mattmoor** [comment]
The downscaler failure looks like a lingering Python 3.12 issue at head with `pykube` as well, and all that's left is Pulumi, which I expect to fail (also b/c of Python 3.12 stuff), so going to TBR this now.

**Pull request merged**